lib/scrape.js

```diff
@@ -233,6 +243,53 @@ async function fetchSEtag(wikidata_id) {
   }
 }
 
+async function fetchInstance(wikidata_id) {
+  const wikiAPIurl = wdk.getEntities(wikidata_id)
+  const page = await fetch(wikiAPIurl)
+  const { entities } = await page.json()
+  const entity = entities[wikidata_id]
+
+  if (!(WIKI_INSTANCE_OF in entity.claims)) {
+    return
+  }
+
+  const instanceClaims = entity.claims[WIKI_INSTANCE_OF]
+
+  if (!instanceClaims) {
+    return
+  }
+
+  const instances = instanceClaims.map(
+    claim => claim.mainsnak.datavalue.value.id
+  )
+
+  return instances
+}
+
+async function fetchGitLabOrg(wikidata_id) {
+  const wikiAPIurl = wdk.getEntities(wikidata_id)
+  const page = await fetch(wikiAPIurl)
+  const { entities } = await page.json()
+  const entity = entities[wikidata_id]
+
+  if (!(WIKI_ACCOUNT_ON in entity.claims)) {
+    return
+  }
+
+  const gitlabClaim = entity.claims[WIKI_ACCOUNT_ON].find(
```

Comment:
this will fail if there is more than one?

it is a good idea to sort the claims by rank ; see https://github.com/coala/gci-leaders/pull/117/files#diff-e7f5b3f014790acdf91e958a37dc5f2cL217
`data.claims[WIKI_SE_TAG].sort`